KachnaOnline.Business/Facades/PushSubscriptionsFacade.cs

```diff
@@ -29,12 +29,26 @@ public class PushSubscriptionsFacade
             _pushSubscriptionsService = pushSubscriptionsService;
         }
 
+        /// <summary>
+        /// Checks whether VAPID keys required for push notifications are present.
+        /// </summary>
+        /// <exception cref="KeysNotAvailableException">When they are not.</exception>
+        private void CheckVapidKeys()
+        {
+            if (string.IsNullOrEmpty(_pushOptions.CurrentValue.PrivateKey) ||
+                string.IsNullOrEmpty(_pushOptions.CurrentValue.PublicKey))
```

Comment:
I think you also have to check for `Subject` – when I first tried it, it threw an exception because it wasn't in the correct format – or does it?